src/app/components/PlayerPanel.ts

```diff
@@ -0,0 +1,132 @@
+import { Player } from '../../Player';
+import { Render } from '../utils/Render';
+import { View } from '../View';
+import { flatten } from 'lodash';
+import { AnimalNames } from '../../Enums/AnimalNamesEnum';
+
+export class PlayerPanel {
+  /**
+   * Creates PlayerPanel based on data given
+   * @param player accepts instance of current player
+   * @param view accepts instance of View componnet
+   */
+  constructor(private player: Player, private view: View) {}
+
+  createPlayerPanel(): HTMLElement {
+    return Render.elementFactory(
+      'div',
+      {
+        className: 'player-panel',
```

Comment:
There are some classes like player__herd and player__data, perhaps we could use player__panel here?